Source/SwiftyDropbox/Platform/SwiftyDropbox_iOS/UtilitiesMobile.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Copyright (c) 2022 Dropbox Inc. All rights reserved.
+//
+
+#if os(iOS)
+
+import Foundation
+import UIKit
+
+@available(iOS 13.0, *)
+extension UIApplication {
+    public func findKeyWindow() -> UIWindow? {
```

Comment:
might be worth making this `internal` on our branch so people aren't accidentally using a SwiftyDropbox extension in random NB code